src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -90,7 +90,7 @@
     public static final String INVALID_ADDRESS_DESC = " " + PREFIX_ADDRESS; // empty string not allowed for addresses
     public static final String INVALID_TAG_DESC = " " + PREFIX_TAG + "hubby*"; // '*' not allowed in tags
 
-    public static final String INVALID_EVENT_DESCRIPTION_DESC = " " + PREFIX_EVENT_DESCRIPTION + "PARTY @ KENTRIDGE";
+    public static final String INVALID_EVENT_DESCRIPTION_DESC = " " + PREFIX_EVENT_DESCRIPTION + "";
```

Comment:
Looking at line 90, empty string don't need the empty string. But we should add comment. I will do the commenting when I fix the freeSchedule test

Reply:
I think we can change the invalid event description to include `/`, would that be fine?